ohsome/test/test_client.py

```diff
@@ -330,22 +301,6 @@ def test_format_bboxes_list(base_client):
     client.elements.count.post(bboxes=bboxes, time=time, filter=fltr)
 
 
-@pytest.mark.vcr
```

Comment:
why as this test removed?

Reply:
we are now testing this on a lower level so we don't need to run the full process